src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -1742,7 +1742,12 @@ void __cdecl main(int argc, char **argv)
     /* Create the main Pool */
     gPool = apxPoolCreate(NULL, 0);
 
-    /* Parse the command line */
+	apxLogOpen(gPool, SO_LOGPATH, SO_LOGPREFIX, SO_LOGROTATE);
+	apxLogLevelSetW(NULL, SO_LOGLEVEL);
+	apxLogWrite(APXLOG_MARK_DEBUG "Apache Commons Daemon procrun (%s %d-bit) logging initialized.",
+		        PRG_VERSION, PRG_BITS);
+
+	/* Parse the command line */
     if ((lpCmdline = apxCmdlineParse(gPool, _options, _commands, _altcmds)) == NULL) {
```

Comment:
It looks like you have used tabs for indents rather than spaces.

Reply:
Weird, I did a cut and paste. Fixed.

Reply:
It would be nice to have "Apache Commons Daemon" in a constant, but that should be done in a separate PR.

Reply:
I know it is pedantic, but you still have a tab before the `/* Parse the command line */` comment

Reply:
It's all good. Details matter :-)

Reply:
Tx +1 to squash and merge (or squash, rebase and merge)